ballerina/sql_client.bal

```diff
@@ -120,10 +122,12 @@ public client class SQLClient {
         return params;
     }
 
-    // TODO: handle composite keys (record types)
-    private function getGetKeyWhereClauses(anydata key) returns sql:ParameterizedQuery|error {
+    private function getGetKeyWhereClauses(anydata... keys) returns sql:ParameterizedQuery|error {
+        keys = <anydata[]>keys[0];
         map<anydata> filter = {};
-        filter[self.keyFields[0]] = key;
+        foreach int i in 0 ..<keys.length() {
```

Comment:
```suggestion
        foreach int i in 0 ..< keys.length() {
```

Reply:
Fixed in [d34ac2d](https://github.com/ballerina-platform/module-ballerina-persist/pull/8/commits/d34ac2d2be4ff9cea04ef2bf6d20ce814fc329f9)